test/630_proxy_dns_test.sh

```diff
@@ -10,8 +10,8 @@ start_suite "Proxy registers containers with dns"
 
 weave_on $HOST1 launch-dns 10.2.254.1/24
 weave_on $HOST1 launch-proxy --with-dns
-docker_proxy_on $HOST1 run -e WEAVE_CIDR=$C2/24 -dt --name=c2 -h $NAME gliderlabs/alpine /bin/sh
-docker_proxy_on $HOST1 run -e WEAVE_CIDR=$C1/24 -dt --name=c1 aanand/docker-dnsutils /bin/sh
+proxy docker_on $HOST1 run -e WEAVE_CIDR=$C2/24 -dt --name=c2 -h $NAME gliderlabs/alpine /bin/sh
```

Comment:
I think "gliderlabs/alpine" and "aanand/docker-dnsutils" should be set as variables in config.sh, so we only have to edit that one place when we change our mind which container to use.